tests/packages/packages/d/package-lock.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Since the package-lock files don't have actual dependencies, it seems that these aren't strictly necessary in source control. Wdyt of excluding them in gitignore?